### ISSUE TITLE
Some doubts about the performance 

### ISSUE THREAD

**Issue by deepcs233** [mit-han-lab/temporal-shift-module#18]
Hi!
Thanks for your interesting work and the source code.
I find that the performance on Sthv1 of TSM with 8-frames and ResNet-50 backbone, efficient test setting is much better than your paper. Have you made any improvements to the original paper?
And can you share the training script on Sthv1  for TSM  with 8-frames and ResNet-50 backbone, which can get the same Top-1 Acc in the GitHub?

Thanks very much

**Comment by tonylins** [comment]
Hi, we basically just adjust the learning rate scheduling. You may follow the hyper parameters mentioned in our paper to reproduce the results.

I will add the training script on Something-something to the README soon.

**Comment by beckybai** [comment]
> Hi!
> Thanks for your interesting work and the source code.
> I find that the performance on Sthv1 of TSM with 8-frames and ResNet-50 backbone, efficient test setting is much better than your paper. Have you made any improvements to the original paper?
> And can you share the training script on Sthv1 for TSM with 8-frames and ResNet-50 backbone, which can get the same Top-1 Acc in the GitHub?
> 
> Thanks very much

May I ask what accuracy you got for TSM and TSN using this code base?
